src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -73,6 +73,23 @@
                 </f:optionalBlock>
             </table>
         </f:block>
+        <f:block>
+            <table>
+                <f:optionalBlock name="slackNotifEnabled"
+                                 title="${%Send notifications to Slack}"
+                                 checked="${descriptor.slackNotifEnabled}"
+                                 inline="true">
```

Comment:
One of the reasons I preferred an optional block was to not use `inline=true`   :) since all slack related data could be stored in a bean and only pollute the settings with one field, sorry if I was unclear on that part.